src/components/TutorCard.tsx

```diff
@@ -76,177 +86,172 @@ const TutorCard: React.FC = () => {
                 </Flex>
                 <CardBody>
                     <Stack spacing="2" textAlign="center">
-                        <Heading size="xs">Patrick Hill</Heading>
+                        <Heading
+                            size="xs"
+                            mb="8px"
+                        >{`${tutor.userId.firstName} ${tutor.userId.lastName}`}</Heading>
                         <Avatar
-                            // src='../assets/patrick.jpg'
                             borderRadius="full"
                             boxSize="100px"
                             mx="auto"
                             style={customAvatarStyle}
                         />
                     </Stack>
                     <Text mt="4" w="100%" fontSize={theme.dashboardButtons.fontSize}>
+                        {tutor.about}
+                    </Text>
+                    {/* <CardBody> */}
+                    {/* <Stack spacing="2" textAlign="center">
+                        <Heading size="xs">Patrick Hill</Heading>
```

Comment:
It seems that this commented-out code is no longer needed. Let's remove it to maintain a code clean.

Reply:
Done